config/envs/development.py

```diff
@@ -19,9 +20,13 @@ class DevelopmentConfig(DefaultConfig):
 
     # for local testing with flask run and USE_LOCAL_DATA = True:
     # USE_LOCAL_DATA = True
+    # if USE_LOCAL_DATA:
+    #     ACCOUNT_STORE_API_HOST = CommonConfig.TEST_ACCOUNT_STORE_API_HOST
+    #     APPLICATION_STORE_API_HOST = CommonConfig.TEST_APPLICATION_STORE_API_HOST
+    #     ASSESSMENT_STORE_API_HOST = CommonConfig.TEST_ASSESSMENT_STORE_API_HOST
+    #     FUND_STORE_API_HOST = CommonConfig.TEST_FUND_STORE_API_HOST
 
     # for testing with docker runner:
-    USE_LOCAL_DATA = False
```

Comment:
did we mean to include these config changes or were they just for local testing?

Reply:
yes I popped them in on purpose, we need a ref to the local data when running locally (without docker) so all that devs need to do is uncomment this code block, and it will call test data.

Reply:
Doesn't CommonConfig already default to these if not env var set? eg. https://github.com/communitiesuk/funding-service-design-utils/blob/main/fsd_utils/config/commonconfig.py#L42

Reply:
It does, but in docker runner if you want to run off local test data we need to restate it otherwise it keeps trying to call the other containers which are picked up in common config (https://github.com/communitiesuk/funding-service-design-docker-runner/blob/main/docker-compose.yml#:~:text=fund%2Dstore%3A8080-,%2D%20APPLICATION_STORE_API_HOST%3Dhttp%3A//application%2Dstore%3A8080,%2D%20ACCOUNT_STORE_API_HOST%3Dhttp%3A//account%2Dstore%3A8080,-ports%3A)

Reply:
using the above you can turn on and off what you want to be mocked and what you dont